### ISSUE TITLE
Uncaught TYPO3 Exception Using $this when not in object context thrown in file /var/www/html/typo3/sysext/core/Classes/Core/Bootstrap.php in line 1095

### ISSUE THREAD

**Issue by wowaTYPO3** [wind-internet/randomdata#8]
Hi, I wanted to test your extension on my local DDEV setup (with a TYPO3 8 LTS), but after the command

`vendor/bin/typo3 randomdata:generate configuration.yaml de_DE`

I got the error message

```
Uncaught TYPO3 Exception Using $this when not in object context
thrown in file /var/www/html/typo3/sysext/core/Classes/Core/Bootstrap.php
in line 1095
```

Setup:
PHP 7.2.32
TYPO3 8.7.32

**Comment by FamousWolf** [comment]
Sorry for the long wait. The problem has been fixed in 1.0.8

**Comment by wowaTYPO3** [comment]
Thanks, works very good! I think, I will record a short video about your extension tomorrow or the day after tomrrow for my youtube channel. Great extension!
